_setup_support.py

```diff
@@ -13,10 +13,12 @@
 import shutil
 import sys
 import time
-from distutils.cmd import Command
 from pathlib import Path
 from subprocess import PIPE, CompletedProcess, run
 
+from setuptools import Command
```

Comment:
In general it is considered good practice to use the `setuptools` version of the available APIs instead of `distutils`'.

Reply:
👍 I had looked for that but just missed it apparently.